config/locales/notification_mailer.en.yml

```diff
@@ -9,3 +9,7 @@ en:
 
       registration_due_dates: "Domains due to be registered tomorrow:"
       bans: "Bans issued yesterday:"
+    daily_auctions_broadcast_email:
+      subject: "Today's .EE domain auction list is available!"
```

Comment:
Use default translation path (example: https://github.com/internetee/registry/blob/master/config/locales/mailers/contact.en.yml). See comments above for details.

Reply:
I agree. But all mailers in `auction_center` app are using same (not perfect) path so I simply followed the pattern. I'm opening a new ticket because moving them all right now is not the scope of this PR.

Reply:
Perhaps we could still start using default paths at least for the things directly connected to this PR?